rudder-lang/tools/tester.sh

```diff
@@ -106,16 +122,15 @@ fi
 
 if [ ${env} = "dev" ] 
 then
-
   ##########################
   # DEVELOPMENT EVIRONMENT #
   ##########################
-  ${cfjson_tester} ncf-to-json --config-file=${config_file} "${technique_path}" "${technique}.json" \
-  && ${rudderc} --config-file=${config_file} --translate -s "${technique}.json" \
-  && ${rudderc} --config-file=${config_file} -s "${technique}.rl" \
-  && ${cfjson_tester} ncf-to-json --config-file="${config_file}" "${technique}.rl.cf" "${technique}.rl.cf.json" \
-  && ${cfjson_tester} compare-json --config-file="${config_file}" "${technique}.json" "${technique}.rl.cf.json" \
-  && ${cfjson_tester} compare-cf --config-file="${config_file}" "${technique_path}" "${technique}.rl.cf"
+  ${cfjson_tester} ncf-to-json --config-file=${config_file} "${technique_path}" "${technique}.json" && status=${status}+1 \
+    && ${rudderc} --config-file=${config_file} --translate -s "${technique}.json" && status=${status}+1 \
+    && ${rudderc} --config-file=${config_file} -s "${technique}.rl" && status=${status}+1 \
+    && ${cfjson_tester} ncf-to-json --config-file="${config_file}" "${technique}.rl.cf" "${technique}.rl.cf.json" && status=${status}+1 \
```

Comment:
status=${status}+1 it not proper shell you must write 
status=$((${status}+1)) or status=$((status+1))

Reply:
fixed